org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -389,6 +389,11 @@ SsoDeveloper:
       {
           "Version": "2012-10-17",
           "Statement": [
+              {
+                  "Effect": "Allow",
+                  "Resource": "arn:aws:iam::*:role/eks-developer-viewer-role-*",
```

Comment:
Currently, I added in a wild card for the AWS account ID here:
`arn:aws:iam::*:role`

Is there a way I can add in a variable here instead? This `eks-developer-viewer-role` is going to exist within the same account that this `SsoDeveloper` role is going to exist, However, I was not sure how I could add that to this `_tasks.yaml` file to add in a template variable here.